src/hazmat/primes.rs

```diff
@@ -223,10 +223,9 @@ pub(crate) const PRIMES_1024: &[U1024] = &[
     ]),
 ];
 
-// A Cunningham chain (of the first kind) is a sequence of `k` numbers
-// such that `n_{k+1} = n{k} * 2 + 1`, and each `n_k` is prime.
-// We provide pairs `(k, n_0)` here.
-
+/// A Cunningham chain (of the first kind) is a sequence of `k` numbers
+/// such that `n_{k+1} = n{k} * 2 + 1`, and each `n_k` is prime.
```

Comment:
Could you fix `n{k}` -> `n_k` as well? Not that we can render TeX in docs.rs now, but that's what it's supposed to be :)